recipes/log.c/all/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+cmake_minimum_required(VERSION 3.16)
+
+project(log.c 
+	    VERSION 0.1.0 
+	    DESCRIPTION "A simple logging library implemented in C99" 
+	    HOMEPAGE_URL "https://github.com/rxi/log.c"
+	    LANGUAGES C
+	)
+
+# Init Conan stuff
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+# Set paths
+include(GNUInstallDirs)
+set(SRC_FOLDER "source_subfolder")
+
+# Library files SHARED or STATIC set by Conan with BUILD_SHARED_LIBS
+add_library(${PROJECT_NAME} ${SRC_FOLDER}/src/log.c ${SRC_FOLDER}/src/log.h)
```

Comment:
Does is support dll's on Windows (MSVC+mingw)?

MSVC requires explicit exporting of symbols.

Reply:
I use it as static lib at Unix and just check it compile at windows as static lib.
If you help, I'll try with mingw

Reply:
It won't work on msvc because the project does not export any symbols:
https://github.com/rxi/log.c/blob/master/src/log.h#L40-L47

The CI will fail for MSVC+shared=True